app/Mcp/Concerns/BuildsResponse.php

```
@@ -180,14 +192,17 @@ protected function actionsForDatabase(string $uuid, ?string $status = null): arr
     {
         $actions = [
             ['tool' => 'get_database', 'args' => ['uuid' => $uuid], 'hint' => 'Full details'],
+            ['tool' => 'list_env_keys', 'args' => ['resource' => 'database', 'uuid' => $uuid], 'hint' => 'Env key names (no values)'],
+            ['tool' => 'list_database_backups', 'args' => ['uuid' => $uuid], 'hint' => 'Backup schedules'],
+            ['tool' => 'list_storages', 'args' => ['resource' => 'database', 'uuid' => $uuid], 'hint' => 'Volumes / file mounts'],
         ];
 
         $s = strtolower((string) $status);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $s. Configured minimum length is 3. (undefined)

(ShortVariable)